packages/aws-cdk-lib/core/test/logical-id.test.ts

```diff
@@ -266,6 +286,19 @@ function generateString(chars: number) {
   }
 }
 
+function generateAlphaNumericString(chars: number) {
```

Comment:
We should use `generateAlphaNumericString` also [here](https://github.com/aws/aws-cdk/blob/26dcc1e11a07d93681145049aa06d80a7d2114b9/packages/aws-cdk-lib/core/test/logical-id.test.ts#L171-L176).